python/hyperon/stdlib.py

```diff
@@ -94,7 +131,8 @@ def type_tokens():
         r"[-+]?\d+(\.\d+)?e[-+]?\d+": lambda token: ValueAtom(float(token), 'Number'),
         "\"[^\"]*\"": lambda token: ValueAtom(str(token[1:-1]), 'String'),
         "\'[^\']\'": lambda token: ValueAtom(Char(token[1]), 'Char'),
-        r"True|False": lambda token: ValueAtom(token == 'True', 'Bool')
+        r"True|False": lambda token: ValueAtom(token == 'True', 'Bool'),
+        r'regex:"[^\"]*"': lambda token: RegexMatchableAtom(token)
```

Comment:
`lambda token: RegexMatchableAtom(token)` could be just `RegexMatchableAtom` I believe. Or you can just have `lambda token: G(RegexMatchableObject(token), 'Regex')` without `RegexMatchableAtom`. I don't see any immediate use of `RegexMatchableAtom` with additional `type_name` and `atom_id`.